import-tags.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+public=''
+
+while getopts c:t:pu: flag
+do
+    case "${flag}" in
+        c) collection="${OPTARG}" ;;
+        t) file="${OPTARG}" ;;
+        p) public="-p" ;;
+        u) user="${OPTARG}"
+    esac
+
+uuid=$(./liquid dockerexec hoover:search ./manage.py getuuid $user | tail -1)
```

Comment:
without `set -o pipefail` the `|tail -1` will make any possible errors in `getuuid` command fail silently, and we'll get an empty string here and no error.

please add `set -o pipefail` at the top of the script, at line 2. 

Also, run some manual tests for what happens when bad username is given; script should halt at this line.